docs/evm_compatibility/overview.md

```diff
@@ -1,63 +1,155 @@
 ---
 title: EVM Compatibility Overview
 proofedDate: na
-iterationBy: na
+iterationBy: HB
 includedInSite: true
 approvedBy: na
-comment: 404 being dealt with on link-fixer branch??
+comments: 
 ---
 
-TL;DR
+## TL;DR
 
-- Apply (most of) Ethereum's standard JSON RPC API methods
-- Direct the calls to Neon's Proxy
-- There are some differences and limitations to consider
+- Apply (most of) Ethereum's standard [JSON RPC API methods](/docs/evm_compatibility/json_rpc_api_methods)
+- Direct the calls to a [Neon RPC](/docs/developing/connect_rpc) via a Proxy Operator
+- Solana and Ethereum differ: consider those differences and the limitations they enforce
 
 ## Introduction
 
-Interacting with Neon EVM is essentially the same as interacting with the [Ethereum Virtual Machine (EVM)](https://ethereum.org/en/developers/docs/evm/).
+Interacting with Neon EVM is essentially the same as interacting with any [Ethereum Virtual Machine (EVM)](https://ethereum.org/en/developers/docs/evm/).
 
-Neon EVM provides a proxy service that accepts [Ethereum's standard RPC API](https://ethereum.github.io/execution-apis/api-documentation/). Your dAPP can apply familiar methods: with your calls directed to the Neon Proxy rather than an Ethereum L1 node.
+Neon EVM provides a proxy service that accepts [Ethereum's standard RPC API methods](https://ethereum.github.io/execution-apis/api-documentation/). 
 
-In this way, Neon EVM provides a seamless developer experience. Note that there are some differences and considerations. 
+> Your dApp can apply familiar methods: with your calls directed to the [Neon Proxy](../about/neon_ecosystem#neon-proxy) rather than an Ethereum L1 node.
+
+In this way, Neon EVM provides a seamless developer experience. Note that there are some differences and considerations due to some signigicant differences between Solana transaction requirements and the equivalent in EVM. 
 
 ## Shared standards and features
 
-Solidity or Vyper smart contracts, standard development and deployment tools and practices can all be applied on Neon EVM with minimal reconfiguration of the code:
+Solidity or Vyper smart contracts, standard development and deployment tools and practices can all be applied on Neon EVM with minimal reconfiguration of the code.
 
 - Neon EVM accounts follow [Ethereum's account standards](https://ethereum.org/en/developers/docs/accounts/) 
 
-> All [Ethereum opcodes](https://www.evm.codes/?fork=merge) are represented verbatim on Neon EVM. 
+>  The majority of [Ethereum OpCodes](https://ethereum.org/en/developers/docs/evm/opcodes) are represented verbatim on Neon EVM. Learn about stacks: 41, 44, 45, and 48 [OpCodes](opcodes).
+
+- Neon EVM supports type 0 / legacy transaction requests
```

Comment:
@anton-lisanin I recall that there was not yet support for EIP-1559, is this still the case?

Reply:
It is still the case. We support only legacy transactions.